core/base/_links.scss

```diff
@@ -4,21 +4,58 @@
 
 
 /**
+ * Base link styles, at the bare minimum a colour is applied for both default
+ * and hover states. There is the option to remove the underline for the
+ * default state and apply on hover state and an option to transition the
+ * colour on hover.
+ *
+ * N.B. a mixin is used to contain the base link styles so that it can be
+ * easily shared with other parts of Scally e.g. Link complex object.
+ */
+
+
+/**
  * Settings.
  */
 
-$link-color:         $color-brand !default;
+// Toggle on/off certain styles and treatments
+$apply-link-underline-on-hover:     true !default;
+
+$apply-link-transition-on-hover:    true !default;
+
+// Colours
+$link-color:                        $color-brand !default;
 
-$link-color-hover:   darken($link-color, 10%) !default;
+$link-color-hover:                  darken($link-color, 10%) !default;
```

Comment:
Yeah that looks good, I've created this: https://github.com/chris-pearce/scally/issues/107. For now I'll leave it in for the sake of getting V2 done but will definitely come back and add those features :+1:.